index.html

```diff
@@ -53,5 +53,257 @@ <h2 class="section-header-subtitle">I’m a Software Developer</h2>
         </ul>
       </section>
     </section>
+
+<section class="whole-middle-section">
+    <section class="recent-work">
+        <h1 class="recent-work-content">
+            My Recent Works
+        </h1>
+
+    </section>
+
+    <section class="section-bg">
+        <div class="bg">
+        </div>
+    </section>
+
+    <section class="multi-post">
+        <h1 class="multipost-content">
+            Multi-Post Stories
+        </h1>
+    </section>
+
+    <div class="paragraph-box">
+        <p class="paragraph">
+            A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.
+        </p>
+        <ul class="languages">
+            <li class="btn ">CSS</li>
+            <li class="btn ">HTML</li>
+            <li class="btn ">Bootstrap</li>
+            <li class="btn ">Ruby</li>
+        </ul>
+        
+        <button class="btn-pink">See Project</button>
```

Comment:
- [ ] Good job on adding and styling the buttons but kindly add the button interactions following [Figma design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=1%3A1471).

Here you have a screenshot from the design for button interactions:

![image](https://user-images.githubusercontent.com/74506933/159077509-e2fe192d-9281-4fac-a420-d7badf1b0a3b.png)

Reply:
@Donard97, is there any property as 'pressed' in css?

Reply:
@Donard97 or were talking about active?